rtc/configuration.md

```diff
@@ -136,143 +136,198 @@ tinymce.init({
 })
 ```
 
-## Optional configuration
+## Recommended configuration
 
-### `rtc_snapshot`
+For the best user experience, {{site.companyname}} recommends including these configuration settings:
 
-Real-time collaboration integrations regularly store the content, eliminating the need for a save button. The {{site.productname}} RTC plugin provides a version number to assist with storing the regular content snapshots. These snapshots are not stored by the {{site.cloudname}} and must be handled by the integrator.
+* [`rtc_server_disconnected`](#rtc_server_disconnected)
+* [`rtc_user_details_provider`](#rtc_user_details_provider)
 
-For any given document ID, the server guarantees the version number will only increase. It can be safely used for conflict resolution. For each document ID and version combination the snapshot content is guaranteed to be identical.
+### `rtc_server_disconnected`
 
-The snapshot callback will be executed at regular intervals with access to the serialized editor content. The content is retrieved through a `getContent` function to reduce CPU load if the callback decides to not use the editor content.
+If the RTC session fails to connect, or is disconnected due to an error, the user will be blocked from editing (using [setProgressState]({{site.baseurl}}/api/tinymce/tinymce.editor/#setprogressstate)) along with an error notification:
+
+![RTC disconnected error example]({{site.baseurl}}/images/rtc-error-example.png "RTC disconnected error example")
 
+The server disconnected callback can be used to provide an integration response to this condition. The function is provided both a reason for the disconnect and a suggested HTML error message to display to the user.
```

Comment:
"The server disconnected callback can be used to provide an integration response to this condition." This reads a bit fluffy, might not even need it?

Reply:
I'm describing the fields. I'll see if I can move the wording down to the field description and it still makes sense.